src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/NativeRuntimeEventSource.Threading.cs

```diff
@@ -504,5 +514,45 @@ public unsafe void ThreadPoolIOPack(NativeOverlapped* nativeOverlapped)
             data[4].Reserved = 0;
             WriteEventCore(59, 5, data);
         }
+
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:UnrecognizedReflectionPattern", Justification = "Parameters to this method are primitive and are trimmer safe")]
```

Comment:
maybe this was already discussed but is `WaitHandleWaitSourceMap` "trimmer safe"?

Reply:
This should be fine. We're doing suppressions like this for enums elsewhere. It's only problematic when this involves recursive graph serialization.